src/utils/secure.filter.ts

```diff
@@ -3,17 +3,27 @@ import { ApplicationData } from "../model";
 import { IsSecureRegisterKey, OverseasEntityKey, Transactionkey } from "../model/data.types.model";
 import { getApplicationData, setExtraData } from "./application.data";
 import { logger } from "./logger";
-import { getUrlWithTransactionIdAndSubmissionId, isRemoveJourney } from "../utils/url";
+import { getUrlWithTransactionIdAndSubmissionId, isRemoveJourney, isRegistrationJourney } from "../utils/url";
 import * as config from "../config";
 import { isActiveFeature } from "./feature.flag";
 import { updateOverseasEntity } from "../service/overseas.entities.service";
 import { Session } from "@companieshouse/node-session-handler";
 
 export const getFilterPage = async (req: Request, res: Response, next: NextFunction, templateName: string, backLinkUrl: string): Promise<void> => {
+
   try {
+
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
-    const appData: ApplicationData = await getApplicationData(req.session);
-    const isRemove: boolean = await isRemoveJourney(req);
+
+    const isRegistration = isRegistrationJourney(req);
```

Comment:
if you often need to check the different journeys in the same place
maybe having a function returning the status of each one would be simpler

{
   isRegistration: true,
   isUpdate: false,
   isRemove: false
}

Reply:
Ok, I see your point. The only issue is that this is temporary code and will be deleted when the REDIS ticket for the Update journey is done. Only the existing `isRemove` check will remain (as before).

Reply:
it was a proposal
if we don't have the case elsewhere in the code, we certainly don't need it

Reply:
Because of a code smell from Sonar, I have refactored the `getApplicationData` piece that is common to both methods into a separate, non-exportable local method called `fetchApplicationData`. But this is also temporary until the Update journey ticket is done.